### DISCUSSION TITLE
Updating Signal-CLI Along with APT Updates?

### DISCUSSION THREAD

**Discussion by muad-dumb** [AsamK/signal-cli#1179] [Q&A]
New to Linux, so apologies if this is basic or a misunderstanding on my part:

My Signal-CLI doesn't update or upgrade when I run my apt commands through CLI, when I'm running my normal package updates. Is there a way to make it do this?

I installed via the tar.gz, just unpacking it into a folder.

**Comment by exquo** [comment]
Because it was not installed with `apt`, it would not be updated by `apt`. If you would like it to be, there is a third-party APT package that you can install here: https://packaging.gitlab.io/signal-cli/installation/standalone. 
In the wiki: https://github.com/AsamK/signal-cli/wiki/Binary-distributions#Debian
